packages/ocamlgraph/ocamlgraph.2.0.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "A generic graph library for OCaml"
+description: "Provides both graph data structures and graph algorithms"
+maintainer: ["filliatr@lri.fr"]
+authors: ["Sylvain Conchon" "Jean-Christophe Filliâtre" "Julien Signoles"]
+license: "LGPL-2.1-only"
+tags: [
+  "graph"
+  "library"
+  "algorithms"
+  "directed graph"
+  "vertice"
+  "edge"
+  "persistent"
+  "imperative"
+]
+homepage: "https://github.com/backtracking/ocamlgraph/"
+bug-reports: "https://github.com/backtracking/ocamlgraph/issues/new"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "stdlib-shims"
+  "dune" {>= "2.0"}
```

Comment:
```suggestion
  "dune" {>= "2.0" & !with-test | >= "2.8"}
```
This fixes the issue with the `graphics` package (which at present means `ocamlgraph.2.0.0` can't be installed with `--with-test` on 4.08 and earlier)

Reply:
done